src/main/java/com/formulasearchengine/mathmlconverters/MathMLConverter.java

```diff
@@ -194,11 +205,18 @@ private void removeAttribute(Node node, String name) {
      */
     Content scanFormulaNode(Element formulaNode) throws Exception {
         // first off, try scanning for mathml nodes directly
+        Element annotationNode = (Element) XMLHelper.getElementB(formulaNode, xPath.compile("//m:annotation-xml"));
+        Boolean encodingMathML;
```

Comment:
The name of the boolean should already indicate that you try for content mathml and not just any encoding.
Also I think, the one-line-if here would still be sleek and readable.